packages/vkui/src/testing/e2e/index.playwright.ts

```diff
@@ -81,6 +83,28 @@ export const test = testBase.extend<VKUITestOptions & InternalVKUITestOptions &
     });
   },
 
+  expectA11yScanResults: async ({ page }, use, testInfo) => {
+    const axeBuilder = () =>
+      new AxeBuilder({ page }).withTags(['wcag2a', 'wcag2aa', 'wcag21a', 'wcag21aa']);
+
+    const accessibilityScanResults = async () => {
+      const results = await axeBuilder()
+        .include('.' + TEST_CLASS_NAMES.CONTENT)
```

Comment:
```suggestion
        .include('#root')
```

Если хотим проверить компонент, то стоит через `#root` его анализировать

`TEST_CLASS_NAMES.CONTENT` это класс, который добавляет `color`

https://github.com/VKCOM/VKUI/blob/36359324e961e532dca248c0603f3d3cddbc605b/packages/vkui/playwright/index.css#L70-L73

В случае с аккордеоном он используется для его контента, чтобы на скриншотах был нормальный цвет текста:

https://github.com/VKCOM/VKUI/blob/36359324e961e532dca248c0603f3d3cddbc605b/packages/vkui/src/components/Accordion/Accordion.e2e-playground.tsx#L30-L33

Reply:
➕ 
Спасибо, пофиксила!